components/speedreader/rust/lib/src/readability/src/scorer.rs

```diff
@@ -491,6 +491,7 @@ pub fn preprocess(mut dom: &mut Sink, handle: Handle) -> bool {
                     local_name!("br") => brs.push(child.clone()),
                     _ => {
                         if brs.len() >= 2 {
+                            brs.pop();
```

Comment:
test?

Reply:
![image](https://user-images.githubusercontent.com/73575789/186154239-da5fc1c5-8cc6-4859-891e-322b10425170.png)

Reply:
Test data updated. Here is visual report [speedreader_report.zip](https://github.com/brave/brave-core/files/9403207/speedreader_report.zip).